weaviate/filters/where.go

```diff
@@ -210,6 +204,112 @@ func (b *WhereBuilder) string() string {
 	return strings.Join(clause, " ")
 }
 
+func (b *WhereBuilder) ToGRPC() *pb.Filters {
+	filters := &pb.Filters{}
+
+	switch b.operator {
+	case Or:
+		filters.Operator = pb.Filters_OPERATOR_OR
+		filters.Filters = operandsToGRPC(b.operands)
+
+	case And:
+		filters.Operator = pb.Filters_OPERATOR_AND
+		filters.Filters = operandsToGRPC(b.operands)
+
+	// TODO aliszka add Not
+
+	default:
+		//nolint:staticcheck // SA1019
```

Comment:
What kind of lint error does this silence?

Reply:
`filters.On` is deprecated, but since client's api allows only to provide path this way, deprecated method have to be used for now.
will add another comment with explanation

Reply:
Hey - `FilterTarget target = 20` is the correct way to provide the target. I am about to remove `On`- could you please switch it over?